include/mega/command.h

```diff
@@ -92,9 +92,19 @@ class MEGA_API Command
     Command();
     virtual ~Command() = default;
 
+    bool static checkError(int64_t &e, JSON &json, ErrorDetails &errorDetails);
+
     MEGA_DEFAULT_COPY_MOVE(Command)
 };
 
+class ErrorDetails
```

Comment:
Maybe we could include the error code inside this class, which could represent the `Error` :thinking: 
Just thinking loud ^

Reply:
In method `bool Command::checkError(int64_t& e, JSON& json, ErrorDetails& errorDetails
Parameter `e` is not always an error. If we want to reduce the number of parameter it's possible to add in `ErrorDetails` but maybe we have to rename the class

Reply:
This has been changed